### PR TITLE
Added: Copy math as TeX commands for exported html

### PR DESCRIPTION
**Pull request by upupming**
Added copy TeX feature 😄 ~
See https://github.com/upupming/katex-copytex for more information.

### CODE REVIEW

**Comment by neilsustc** [comment]
Interesting. I played around and felt there are two problems.

1. don't work when you select some text with inline math inside.
   image
2. don't work on Microsoft Edge

File: src/print.ts [hunk 1]
```diff
@@ -194,17 +194,21 @@ function readCss(fileName: string) {
     }
 }
 
-function getStyles(uri: vscode.Uri) {
+function getScriptsAndStyles(uri: vscode.Uri) {
     const katexCss = '<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.10.0/dist/katex.min.css" integrity="sha384-9eLZqc9ds8eNjO3TmqPeYcDj8n+Qfa4nuSiGYa6DjLNcv9BtN69ZIulL9+8CqC9Y" crossorigin="anonymous">';
     const markdownCss = '<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/markdown.css">';
-    const highlightCss = '<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/highlight.css">'
+    const highlightCss = '<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/highlight.css">';
+    const copyTeXJs = '<script src="https://cdn.jsdelivr.net/npm/katex-copytex@latest/dist/katex-copytex.min.js"></script>';
```
Comment (neilsustc) [inline src/print.ts]: I would rather not mix JS and CSS in one function. Directly putting this line after line 138 would be fine.

**Comment by neilsustc** [comment]
Thanks a lot!

I will release a new version after fixing #323, probably this weekend. And feel free to investigate if you are interested 😆.

**Comment by Lemmingh** [comment]
@upupming

May I know the difference between <https://github.com/upupming/katex-copytex> and <https://github.com/KaTeX/KaTeX/tree/master/contrib/copy-tex>?

Thank you.

**Comment by upupming** [comment]
@Lemmingh No major difference here, we can just use that from katex repo.

**Pull request merged**